src/System Application/Test/Guided Experience/src/DuplicatedGuidedExpItem.Codeunit.al

```diff
@@ -0,0 +1,128 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace System.Test.Environment.Configuration;
+
+using System.Environment.Configuration;
+using System.TestLibraries.Security.AccessControl;
+using System.TestLibraries.Utilities;
+
+codeunit 132599 "Duplicated Guided Exp. Item"
+{
+    Subtype = Test;
+    Permissions = tabledata "Guided Experience Item" = rimd;
+
+    var
+        Assert: Codeunit "Library Assert";
+        PermissionsMock: Codeunit "Permissions Mock";
+        DummyGuidedExperienceCode1Lbl: Label 'Dummy Guided Experience Code 1', Locked = true;
+        DummyGuidedExperienceCode2Lbl: Label 'Dummy Guided Experience Code 2', Locked = true;
+
+    [Test]
+    procedure TestInsertDuplicatedGuidedExperienceItems()
+    var
+        GuidedExperienceItem: Record "Guided Experience Item";
+        GuidedExperience: Codeunit "Guided Experience";
+        Limit, Counter : Integer;
+    begin
+        // [SCENARIO] Insert multiple Guided Experience Items with the same Code
+        PermissionsMock.Set('Guided Exp Edit');
+
+        GuidedExperienceItem.DeleteAll();
+
+        Assert.AreEqual(0, GuidedExperienceItem.Count(), 'No records should exists in the Guided Experience Item table before the test starts.');
```

Comment:
Labels

Reply:
Not sure what this the convention here tbh, those messages don't need to be translated. I think it is ok for now